### ISSUE TITLE
proot error while trying to run xfce4 in ubuntu-20.04

### ISSUE THREAD

**Issue by danyael031** [termux/proot#133]
I'm trying to start a vnc server with xfce inside ubuntu-20.04 with poot-distro. If i try to do something, like open the file explorer, i get this error, ubuntu closes and termux get buggy:

```
root@localhost:~# vncserver :1

New 'X' desktop is localhost:1

Starting applications specified in /etc/X11/Xvnc-session
Log file is /root/.vnc/localhost:1.log

root@localhost:~# ./extension/sysvipc/sysvipc.c:39: int sysvipc_syscall_common(Tracee *, struct SysVIpcConfig *, _Bool): assertion "config->wait_state == WSTATE_NOT_WAITING" failed
proot warning: signal 6 received from process 18147
$ 
```
i'm using tigervnc and xfce4 was installed with flag --no-install-recommends. Inside ~/.vnc/xstartup i use this:
``` sh
#!/bin/sh
dbus-launch startxfce4
```

the commands that i use after login to ubuntu-20.04 with proot-distro are these:

```sh
apt update && apt upgrade
apt install dbus-x11
apt install --no-install-recommends tigervnc-standalone-server xfce4
apt install tigervnc-common
vncserver -localhost
vncserver -kill :1

vim .vnc/xstartup

# +++ #!/bin/sh
# +++ dbus-launch startxfce4

vncserver :1

# Output:
# New 'localhost.localdomain:1 (root)' desktop at :1 on machine localhost.localdomain
#
# Starting applications specified in /root/.vnc/xstartup
# Log file is /root/.vnc/localhost.localdomain:1.log
#
#Use xtigervncviewer -SecurityTypes VncAuth -passwd /root/.vnc/passwd :1 to connect to the VNC server.


# If i try to run something like open the file explorer i get this:
#root@localhost:~# ./extension/sysvipc/sysvipc.c:39: int sysvipc_syscall_common(Tracee *, struct SysVIpcConfig *, _Bool): #assertion "config->wait_state == WSTATE_NOT_WAITING" failed
#proot warning: signal 6 received from process 29691
#$ 

```

If i use the flag  `--no-sysvipc` in `proot-distro login` the error doesn't happen. 

Termux info output (maybe useful):

```
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://dl.bintray.com/termux/termux-packages-24/ stable main
# game-repo (sources.list.d/game.list)
deb https://dl.bintray.com/grimler/game-packages-24 games stable
# science-repo (sources.list.d/science.list)
deb https://dl.bintray.com/grimler/science-packages-24 science stable
# unstable-repo (sources.list.d/unstable.list)
deb https://dl.bintray.com/xeffyr/unstable-packages unstable main
# x11-repo (sources.list.d/x11.list)
deb https://dl.bintray.com/xeffyr/x11-packages x11 main
Updatable packages:
All packages up to date
Android version:
10
Kernel build information:
Linux localhost 4.14.113-user+ #1 SMP PREEMPT Fri Sep 18 01:04:51 CDT 2020 aarch64 Android
Device manufacturer:
motorola
Device model:
motorola one action
```

**Comment by WMCB-Tech** [comment]
Yeah. i encountered the same. but restarting over and over again helps

**Comment by michalbednarski** [comment]
I think I have this fixed, update should soon be available in Termux apt repo (as proot 5.1.107-33)

**Comment by danyael031** [comment]
The problem was fixed in prrot 5.1.107-33. Thanks, @michalbednarski !
